deployer/templates/fluentd.yaml

```diff
@@ -159,6 +159,10 @@ objects:
               value: ${USE_MUX_CLIENT}
             - name: "MUX_ALLOW_EXTERNAL"
               value: ${MUX_ALLOW_EXTERNAL}
+            - name: "BUFFER_QUEUE_LIMIT"
```

Comment:
The work is going to be in the various test-*.sh scripts that use the logging-fluentd-template.  They should modify the daemonset directly.